vision_srv_server_node:
  ros__parameters:
    service_name: "riconosci_oggetto"
